annotate-equations.tex

```diff
@@ -263,7 +287,7 @@ \subsection{Line breaks within annotations}
 \end{LTXexample}
 \noindent
 %
-Here is a manual work-around:
+Here is a manual work-around: (obsolete)
```

Comment:
I don't think it's helpful to show the old work-around - could you just delete it instead? All it needs to say here is that using `\\` within the text to get a line break works just fine.:)